metacat-thrift/src/main/java/com/netflix/metacat/thrift/CatalogThriftHiveMetastore.java

```diff
@@ -295,7 +312,15 @@ public AddPartitionsResult add_partitions_req(final AddPartitionsRequest request
      */
     @Override
     public void alter_database(final String dbname, final Database db) throws TException {
-        throw unimplemented("alter_database", new Object[]{dbname, db});
+        requestWrapper("update_database", new Object[]{db}, () -> {
+            if (dbname == null || db == null) {
+                throw new InvalidInputException("Invalid database request");
+            }
+            final DatabaseCreateRequestDto dto = new DatabaseCreateRequestDto();
+            dto.setMetadata(db.getParameters());
```

Comment:
I would prefer to use builder pattern, instead of new an object and set the property.

Reply:
Will do.